site2/website/release-notes.md

```diff
@@ -3,6 +3,15 @@
 
 ### 2.8.0 &mdash; 2021-06-12 <a id=“2.8.0”></a>
 
+### Update notice
+
+Due to a breaking change in the Schema API it may happen that some Pulsar Functions or Pulsar IO Connector fail to work, 
+throwing an `IncompatibleClassChangeError`.
+In this case you have to rebuild your Function using Apache Pulsar 2.8.0 as dependency and redeploy it.
+If you are running on Kubernetes you can temporarily let the Functions Worker pod run with a previous version of Pulsar
```

Comment:
From my understanding, for Kubernetes users, we need to take into account the runtime which users using. If users use thread or process runtime, then they should keep functions worker pod with a previous version of Pulsar. If users use Kubernetes runtime, they should keep the `functionRuntimeFactoryConfigs.functionDockerImages` or `functionRuntimeFactoryConfigs.pulsarDockerImageName` with previous version of Pulsar in `functions_worker.yml`.

Reply:
even this is technically possible,
it is not simple to configure this parameter in a multi tenant environment, it is practically impossible, because you have to take care of the placement of the functions for each tenant.

Also we do not guarantee (it is not stated anywhere and there is no testcase) that the 'function worke' is able to work with a more recent version of the broker or of the other components. 
In practice if you upgrade from 2.7.x to 2.8.x you can keep the function pods at version Pulsar 2.7.x, but this is not something that I would recommend to the public audience, as it needs some advanced knowledge of Pulsar internals

if the default is 'apachepulsar/pulsar:latest'  this is kind of a bad problem, because in this case everything may break without notice.
if you are using the official Apache Pulsar Helm Chart we are setting properly the version 

does this make sense to you @freeznet ?